apps/site/pages/docs/themes/overview.mdx

```diff
@@ -46,37 +46,40 @@ By default, every FastStore project comes with the [Brandless theme](/docs/themi
 <br></br>
 
 ```scss filename="src/themes/my-theme.scss"
-.theme {
-  // --------------------------------------------------------
-  // Colors (Branding Core)
-  // --------------------------------------------------------
-
-  // --------------------------------------------------------
-  // Typography (Branding Core)
-  // --------------------------------------------------------
-
-  // --------------------------------------------------------
-  // Spacing (UI Essentials)
-  // --------------------------------------------------------
-
-  // --------------------------------------------------------
-  // Grid & Layout (UI Essentials)
-  // --------------------------------------------------------
-
-  // --------------------------------------------------------
-  // Interactive Controls (UI Essentials)
-  // --------------------------------------------------------
-
-  // --------------------------------------------------------
-  // Refinements
-  // --------------------------------------------------------
-
-  // --------------------------------------------------------
-  // FS UI Components
-  // --------------------------------------------------------
-    .section {
-      // Add here the customizations for FastStore UI components.
-    }
+
+@layer theme {
```

Comment:
This was the easiest way that I thougth to add this layer to the priority list.
Does anyone have a better idea, given our current structure?

Reply:
I still need to update this on our documentation!